tools/runqslower.py

```diff
@@ -109,9 +113,12 @@
     if (prev->state == TASK_RUNNING) {
         tgid = prev->tgid;
         pid = prev->pid;
-        if (!(FILTER_PID || pid == 0)) {
-            u64 ts = bpf_ktime_get_ns();
-            start.update(&pid, &ts);
+        u64 ts = bpf_ktime_get_ns();
+        if (pid != 0) {
+            if ( (!(FILTER_PID) && FILTER_TGID == 0) ||
+                 (!(FILTER_TGID) && FILTER_PID == 0)) {
```

Comment:
Can the above be simplied to !(FILTER_PID) && !(FILTER_TGID)?
The same for below?